cmd/kubelet/app/server.go

```diff
@@ -493,7 +502,7 @@ func SimpleKubelet(client *clientset.Clientset,
 		CPUCFSQuota:               true,
 		DiskSpacePolicy:           diskSpacePolicy,
 		DockerClient:              dockerClient,
-		DockerDaemonContainer:     "/docker-daemon",
+		RuntimeContainer:          "/docker-daemon",
```

Comment:
Can we set this value as empty string?  I don't want integration tests to try and move my docker daemon...